.github/workflows/build_ecs_prefixed.yaml

```diff
@@ -38,7 +38,7 @@ jobs:
             -
                 uses: shivammathur/setup-php@v2
                 with:
-                    php-version: 8.0
+                    php-version: 8.1
```

Comment:
php-scoper was not support php 8.1 yet, so on scoping after downgrade, php-version need to be back to latest php 8.0, like in rector 

https://github.com/rectorphp/rector-src/blob/7adde5807cf0c9531a9e3a193b6c318307bfba6a/.github/workflows/build_scoped_rector.yaml#L61-L66

Reply:
Should be better now: https://github.com/humbug/php-scoper/releases/tag/0.17.0

Reply:
Let's ask there :)

Reply:
Ok, let's try on rector , currently rector using pinned version 0.14:

```diff
-wget https://github.com/humbug/php-scoper/releases/download/0.14.0/php-scoper.phar -N --no-verbose
+wget https://github.com/humbug/php-scoper/releases/download/0.17.0/php-scoper.phar -N --no-verbose
```

Reply:
@TomasVotruba it needs to be updated to 0.17 then

https://github.com/symplify/symplify/blob/dbe20966aa4edf210bd28deb6646cfe452c4b8f2/packages/easy-ci/build/build-easy-ci-scoped.sh#L34

Reply:
Various files that need update php-scoper to 0.17

https://github.com/symplify/symplify/blob/dbe20966aa4edf210bd28deb6646cfe452c4b8f2/packages/easy-ci/build/build-easy-ci-scoped.sh#L34

https://github.com/symplify/symplify/blob/c17ed4cecca4bce9049c6839389a73374dc40b40/packages/monorepo-builder/build/build-monorepo-builder-scoped.sh#L34

https://github.com/symplify/symplify/blob/c16b4893d1e16f44c9aa5c0440b86afb5dd06d1e/packages/config-transformer/build/build-config-transformer-scoped.sh#L34